casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -422,8 +422,15 @@ object BlockAPI {
         dag        <- MultiParentCasper[F].blockDag
         maybeBlock <- getBlock[F](hash, dag)
         blockInfo <- maybeBlock match {
-                      case Some(block) =>
-                        getFullBlockInfo[F](block).map(_.asRight[Error])
+                      case Some(block) => {
+                        val blockAdded = dag.contains(block.blockHash)
```

Comment:
I agree with your changes, but they should be addition to #2890.

Please fix `getBlock` (private). Remove Dag parameter because it's not used, the name will clash with public overload so `getBlockFromStore` would be more appropriate.

Reply:
Fixed in https://github.com/rchain/rchain/pull/2890/commits/efd5d419141ea3b5569bd3a11490e622c1b93c82